introc/c_memory_model.tex

```diff
@@ -56,37 +55,50 @@ \subsection{Structs}
 // Let's convert to a c string
 string* bhuvan_name;
 bhuvan_name = malloc(sizeof(string) + length+1);
-/*
-Currently, our memory looks like this with junk in those black spaces
-                ___ ___ ___ ___ ___ ___ ___ ___ ___ ___ ___
- bhuvan_name = |___|___|___|___|___|___|___|___|___|___|___|
+\end{lstlisting}
+
+Currently, our memory looks like the following image. There is nothing in those boxes
 
-*/
+\begin{figure}[H]
```

Comment:
I don't know where we cover endianness, but we should mention that the figures are in big-endian and the addresses increase from left to right.